dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties

```diff
@@ -1,4 +1,5 @@
 otel.exporter.jaeger.endpoint=http://localhost:14250
 otel.traces.exporter=jaeger
 otel.exporter.jaeger.timeout=10000
-otel.experimental.sdk.enabled=true
\ No newline at end of file
+otel.sdk.disabled=false
+otel.service.name=fat_test
```

Comment:
This change doesn't seem to be related to the rest of the PR.

Was this just something which was forgotten in a previous PR?

Reply:
Yes the FAT still used the old config property

Reply:
ok